lib/src/features/feature_renderer.dart

```diff
@@ -8,9 +12,89 @@ import 'polygon_renderer.dart';
 import 'symbol_line_renderer.dart';
 import 'symbol_point_renderer.dart';
 
+class FeatureRendererContext implements Context {
+  FeatureRendererContext(this._context, this.pixelsPerTileUnit);
+
+  final Context _context;
+  final double pixelsPerTileUnit;
+
+  @override
+  LabelSpace get labelSpace => _context.labelSpace;
```

Comment:
It's unfortunate that we need all of this boilerplate so that we can pass around `pixelsPerTileUnilt`. Would it make sense to split out a separate class, e.g. `PixelSpaceMapper` that encapsulates the new logic, and make it a member of `Context`?

Reply:
Good idea!

Reply:
Done